jdbc/src/main/scala/zio/sql/transaction.scala

```diff
@@ -0,0 +1,66 @@
+package zio.sql
+
+import zio.{ Cause, ZIO }
+
+trait TransactionModule { self: SelectModule with DeleteModule with UpdateModule =>
+
+  import Transaction._
+
+  sealed trait Transaction[-R, +E, +A] { self =>
```

Comment:
Do you think it makes sense to introduce the same pattern as with all ZIO-like effects - `ZTransation[R, E, A]` and an alias `Transaction[E, A]`?

Reply:
I will resolve it in next PR